platform/android/export/export_plugin.cpp

```diff
@@ -2272,58 +2256,15 @@ bool EditorExportPlatformAndroid::can_export(const Ref<EditorExportPreset> &p_pr
 		}
 	}
 
-	if (int(p_preset->get("custom_build/export_format")) == EXPORT_FORMAT_AAB &&
-			!custom_build_enabled) {
-		valid = false;
-		err += TTR("\"Export AAB\" is only valid when \"Use Custom Build\" is enabled.");
-		err += "\n";
-	}
-
 	// Check the min sdk version.
 	String min_sdk_str = p_preset->get("custom_build/min_sdk");
+	// TODO: Actually parse this and upstream the fix.
```

Comment:
I actually misunderstood the original logic (it's correct), so I should revert these changes before release.